GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSParachuteSpare.cfg

```diff
@@ -0,0 +1,168 @@
+
+PART
+{
+	name = ROC-SoyuzMSParachuteSpare
+	module = Part
+	author = DECQ
+
+	RSSROConfig = True
+
+	//	============================================================================
+	//	Model and Dimensions
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROCapsules/Assets/DECQ/SoyuzMS/P_SPARE
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.0
+
+	node_stack_top = -0.5, 0.5, 0.5, 0.0, 1.0, 0.0, 1
+	node_attach = 0.0, 0.0, 0.0, 0.0, -1.0, 0.0
+
+	CoPOffset = 0, 0.6, 0.04
+
+	attachRules = 1,0,0,0,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Utility
+	subcategory = 0
+	
+	title = Soyuz TM/TMA/MS Emergency Parachute
+	manufacturer = RKK Energia
+	description = A fast-opening emergency parachute for the Soyuz Descent Module. It is used after the abort sequence and it also serves as a backup for the main parachute.
+	tags = soyuz ms, DECQ
+
+	bulkheadProfiles = size1, srf
+
+	sound_parachute_open = activate
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 0.015
+	dragModelType = default
+	maximum_drag = 0.25
+	minimum_drag = 0.25
+	angularDrag = 0.5
+	crashTolerance = 10
+	breakingForce = 250
+	breakingTorque = 250
+	thermalMassModifier = 1.0
+	skinMassPerArea = 4
+	bodyLiftMultiplier = 0
+	stagingIcon = PARACHUTES
+	stageOffset = 1
+	childStageOffset = 1
+	buoyancyUseCubeNamed = PACKED
+
+	//	============================================================================
+	//	Animations and Textures
+	//	============================================================================
+
+	EFFECTS
+	{
+		rcpredeploy
+		{	AUDIO
+			{	channel = Ship
+				clip = sound_parachute_open
+				volume = 1
+			}
+		}
+		rcdeploy
+		{
+			AUDIO
+			{	channel = Ship
+				clip = sound_parachute_single
+				volume = 1
+			}
+		}
+		rccut
+		{	AUDIO
+			{	channel = Ship
+				clip = RealChute/Sounds/sound_parachute_cut
+				volume = 1
+			}
+		}
+		rcrepack
+		{	AUDIO
+			{	channel = Ship
+				clip = RealChute/Sounds/sound_parachute_repack
+				volume = 1
+			}
+		}
+	}
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = RealChuteModule
```

Comment:
:NEEDS[RealChute]

Reply:
Well considering Realchute is a requirement for RO is it required?